doc/qemu.md

```diff
@@ -0,0 +1,189 @@
+# OBuilder's QEMU Sandbox
+
+This backend should work with any OS which can be booted in QEMU and
+which can provide an SSH interface.
+
+# Base Images
+
+`obuilder` requires a base image aka a root file system, to use as
+the basis of the container which it creates.  For example the `runc`
+backend extracts the root file system from a Docker base image, while
+a FreeBSD `jail` uses a FreeBSD installation on a ZFS volume.  `qemu`
+requires virtual _hard disks_ with the operating system preinstalled.
+
+In order to use the the [QEMU](https://www.qemu.org) backend with
+`obuilder` base images need to be created. A [Makefile](../qemu/Makefile)
+is provided which builds several base images:
+
+- ubuntu-noble-amd64-ocaml-4.14.2.qcow2
+- ubuntu-noble-amd64-ocaml-5.2.0.qcow2
+- ubuntu-noble-riscv64-ocaml-4.14.2.qcow2
+- ubuntu-noble-riscv64-ocaml-5.2.0.qcow2
+- openbsd-67-amd64-ocaml-4.14.2.qcow2
+- openbsd-67-amd64-ocaml-5.2.0.qcow2
+- windows-server-2022-amd64-ocaml-4.14.2.qcow2
+- windows-server-2022-amd64-ocaml-5.2.0.qcow2
+
+The base images build are build using `make ubuntu`, `make windows` or
+`make openbsd`.  The builds are unattended builds requiring no manual
+intervention. Cloud Init is used on Ubuntu, `autounattend.xml` on Windows
+and `autoinstall` on OpenBSD.
+
+# Operation
+
+A spec references the required base image using the `from` directive,
+then runs whatever commands are required.  An trivial to install the
+`tar` package from opam on Windows example is given below.
```

Comment:
I think this is still missing a word or two, I suspect from a suggestion accepting clash. Maybe these are the right ones to add?

```suggestion
then runs whatever commands are required.  A trivial example 
showing how to install the `tar` package from opam on Windows 
is given below.
```